Description: Remove intersphinx extension
 This is doing network access during build.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2019-09-14

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -26,7 +26,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx"]
+extensions = ["sphinx.ext.autodoc"]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
